packages/@lwc/engine/src/framework/restrictions.ts

```diff
@@ -352,8 +351,7 @@ function getCustomElementRestrictionsDescriptors(
                 // programmatically into a lighting element node
                 if (!isUndefined(options)) {
                     logError(
-                        'The `addEventListener` method in `LightningElement` does not support any options.',
-                        this
+                        'The `addEventListener` method in `LightningElement` does not support any options.'
```

Comment:
is the assumption here that because these are descriptors that will throw the error inline, they don't need to have the component stack associated to them? I see many calls to `logError()` in this file that are effectively loosing the ability to display the stack, the question is why?

Reply:
The `logError` used to receive an `Element` to recreate the call stack and traverse the DOM tree to recreate the call stack. Now, `logError` receives a `VM` and climb up using the `owner` chain. The main issue here is that we don't know if the `this` is a standard `Node` coming from the template or a `Node` with an associated `VM`.

For example, all the restrictions related to portals don't have an associated `VM`, there is nothing that we can do in this case, except traversing the DOM tree to find the owner VM (which is what we want to avoid with this PR).